### PR TITLE
Hebrew Translation

### PR DESCRIPTION
**Pull request by orencohendev**
Added most no-brainer Hebrew translations but left the more complicated ones to people better equipped to translate them. Better something than nothing at all.

Also, Hebrew is an RTL language, so exclamation points and dots at the end of a sentence should appear at the leftmost part, but the translation files are LTR, as all JSON files are. Just putting it out there. This should be considered when using an RTL language translation.

### CODE REVIEW

**Comment by bystac** [comment]
Good attempt, Oren,

Unfortunately, without proper direction support (RTL/LTR) and the unfinished translation in this PR, it is unlikely that it will be merged.

I would like to ask if you have gone over the [mega i18l thread](https://github.com/TryGhost/Ghost/pull/8437)? If so, please add your comments or point me to any discussions about the RTL topic.

The Arabic and Hebrew communities would greatly appreciate RTL support. I'm considering tackling these challenges, but I first need to understand the history and decisions made on this topic in the past. If any core devs can share more details, it would be much appreciated.

**Comment by orencohendev** [comment]
I'm on it! Thanks for the update

**Comment by cathysarisky** [comment]
> I'm on it! Thanks for the update

Excellent! :)  I did the RTL adjustments for portal and search as an LTR-only reader, so if you see anything not right, please let me know.

There are some additional files for translation, if you'd like to pick those up also. :)  (Same folder.)

**Comment by orencohendev** [comment]
I pushed the rest of the translations. Anybody who wants to help check it is welcome. I'll install Ghost locally from the branch after work tomorrow to see how it looks when choosing Hebrew.

**Comment by cathysarisky** [comment]
Great.  To get past automated testing (and to see your changes!), you'll need to also edit https://github.com/TryGhost/Ghost/blob/main/ghost/i18n/lib/i18n.js to add hebrew (he) to the list.  

If you're up for more translating, there are two more files to do.  If you have a development environment set up, you can run 'yarn translate' in Ghost/ghost/i18n, which will generate them for you (after you edit the locale into i18n.js).  If you decide not to set up a development environment, you can instead copy the two missing files (search.json and comments.json) from the locales/en folder over to locales/he and then make your edits on the new copy in the he folder.

**Comment by orencohendev** [comment]
> Great. To get past automated testing (and to see your changes!), you'll need to also edit https://github.com/TryGhost/Ghost/blob/main/ghost/i18n/lib/i18n.js to add hebrew (he) to the list.
> 
> If you're up for more translating, there are two more files to do. If you have a development environment set up, you can run 'yarn translate' in Ghost/ghost/i18n, which will generate them for you (after you edit the locale into i18n.js). If you decide not to set up a development environment, you can instead copy the two missing files (search.json and comments.json) from the locales/en folder over to locales/he and then make your edits on the new copy in the he folder.

Thanks! Added he to the list on the i18n.js file. I'll do the other two translations when I have a moment for it after work :)

**Review by cathysarisky — APPROVED** [review]
Ready to merge!  (Given the age of the PR, let's get this merged and then you can add search and newsletter and comments in another PR.)

**Comment by orencohendev** [comment]
Okay great! Once the CI finishes, I'll merge this and will open a different PR for the missing files. Been a little distracted to finish this XD

File: ghost/i18n/locales/he/portal.json
Comment (cathysarisky) [inline ghost/i18n/locales/he/portal.json:1]: Some lines seem to be missing in this file.  Intentional?

**Review by cathysarisky — CHANGES_REQUESTED** [review]
It looks like there are some lines missing from the portal file. I /think/ that's going to lead to a test failure.

File: ghost/i18n/locales/he/portal.json
Comment (orencohendev) [reply ghost/i18n/locales/he/portal.json:1]: Nope, it's probably a rebase oversight. I will fix it and push a new version of the portal file. Thanks for noticing!

**Comment by orencohendev** [comment]
I reviewed all files, and all Hebrew translations were updated at the time of my commit. Let's ensure the CI works before more commits are pushed into the main branch, which could potentially render my translations incomplete. When there are more lines to translate, I'll create a new PR.

**Review by cathysarisky — APPROVED** [review]
Looks good to me! 
Please let me know if you see any RTL issues once these translations are available.  I did the RTL work for search and portal, but as an LTR reader, I might have missed something.

**Comment by orencohendev** [comment]
Seems like the pipeline is failing because of this:

```
Some translations was updated and failOnUpdate option is enabled.
```

Is there anything I need to change in this PR for this to be automatically merged?

**Pull request merged**
